new-components/pkg/test/util/fake_clock.go

```diff
@@ -0,0 +1,11 @@
+package testutil
+
+import (
+	"time"
+
+	"github.com/jonboulle/clockwork"
+)
+
+func FakeClock() clockwork.FakeClock {
+	return clockwork.NewFakeClockAt(time.Date(2024, 11, 1, 0, 0, 0, 0, time.UTC))
```

Comment:
I'd avoid doing this as a static date can be harmful, I think one can simply take care of this in a test setup :)